cardano-cli/test/cli/address-key-hash/run

```diff
@@ -12,12 +12,13 @@ setup_data_dir "${testname}"
 
 error=0
 
+assert_file_exists "${DATA_DIR}/address.vkey"
```

Comment:
This test was marked as passed when I ran it - **however** it was failing in reality - because there was a file inside test dir with name `data` and it should be a file with name `address.vkey`  inside data dir: `data\address.vkey`

Reply:
Thanks for catching that. I don;t think we need an assert that input data exists but we should fix it whenever we find it to be incorrect. Now that ists fixed we should remove the assertion that it exists.